templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml

```diff
@@ -13,7 +13,8 @@ contents:
     spec:
       initContainers:
       - name: discovery
-        image: "{{.Images.setupEtcdEnv}}"
+        image: "{{.Images.machineConfigOperator}}"
```

Comment:
so there's an issue here that is still failing.  I thought I found a fix by subbing MachineConfigOperator -> machineConfigOperator I did this bc I noticed the image referenced was setupEtcdEnv and kubeClientAgentImage which is from https://github.com/openshift/machine-config-operator/blob/master/install/0000_80_machine-config-operator_02_images.configmap.yaml

Reply:
is this image somehow referenced _before_  it has a value? (since this is in a different place than the other mc* images)